specs/consensus.md

```diff
@@ -585,7 +586,7 @@ Apply the following to the state:
 state.accounts[sender].nonce += 1
 state.accounts[sender].balance -= totalCost(tx.amount, bytesPaid)
 
-state.activeValidatorSet[block.header.proposerAddress].pendingRewards += tipCost(bytesPaid)
+state.activeValidatorSet.proposerBlockReward += tipCost(bytesPaid)
 ```
 
 #### Begin Block
```

Comment:
This spec assumes these computations will be done every block. Why not epoch it?

Reply:
I guess this is because this is how it works in the SDK currently. While we want epochs on the long run I'm against putting more effort into this now. It is "just" an optimization (yeah, a really cool one but I think there are higher prio ones).

Reply:
https://github.com/cosmos/cosmos-sdk/discussions/8328 😉 Should land in the sdk before launch. May be worth leaving a note that epochs are preferred.

Reply:
Nice! Thanks for bringing this to our attention.

Reply:
The only real benefit to epochs for our usecase is reducing light client overhead since fewer block headers need to be downloaded. I'll probably be important long-term, but it's just an optimization and we can worry about those later.